examples/length/length.bal

```diff
@@ -48,10 +48,10 @@ public function main() {
     length = x1.*.elements().length();
     io:println("XML child elements size: ", length);
 
-    // This creates a tuple with two members.
+    // This creates a `tuple` with two members.
```

Comment:
IINM, we shouldn't be using `` with tuple since it is not a keyword or a variable name.